src/main/java/Calculator.java

```diff
@@ -0,0 +1,18 @@
+
+public class Calculator {
+    public static MyNumber calc(String text) {
+        if (text == null || text.isEmpty()) {
+            throw new IllegalArgumentException("문자열이 비었습니다.");
+        }
+
+        String[] split = text.split(" ");
+        MyNumber first = new MyNumber(split[0]);
+
+        for (int i = 1; i < split.length; i+=2) {
```

Comment:
```suggestion
        for (int i = 1; i < split.length; i += 2) {
```
포맷팅을 습관하는걸 추천드립니다ㅎㅎ
(자동 포맷팅을 사용하는 방법도 있습니다)